frontend/Controls/+mobile/Header.qml

```diff
@@ -45,6 +45,7 @@ Item {
             }
         }
     }
+    /**
```

Comment:
Can this be removed?

Reply:
preferably not, it's for reference as I build out, will be cut/pasted someplace else